coldfront/core/allocation/views.py

```diff
@@ -1979,8 +1985,21 @@ def get(self, request, *args, **kwargs):
                     'user__email', flat=True
                 ))
 
-            send_email_template(
-                subject, template, template_context, sender, receiver_list)
+            receiver_names = list(receiver_list.values_list('user__first_name', flat=True))
```

Comment:
Only one user's request is getting denied, but other users (managers, PIs) are also getting an email personalized to them saying "Your cluster access request under project ... and allocation ... has been denied.", which is confusing.

I'd think it'd be more appropriate to send only one email, where the first name is that of the denied requester, but relevant managers and PIs are "CC'ed". (They're not technically CC'ed--it might be a separate issue to add these users to a CC list rather than to the main recipients list--but they receive the email as a courtesy.)

Reply:
I have changed it now so that all the `secondary receivers` get cc'ed. Only caveat I can think of is what if multiple people are `primary receivers`, then the `secondary receivers` will receive multiple emails. It Shouldn't be a problem in this case, just pointing it out since we dont have that case implemented yet.

Reply:
Good thinking. You might want to investigate what Django's `send_mail` method does in that case with some test emails. If emails are sent twice, we'll want to remove duplicates from the secondary list.

I mentioned only this email, but the other emails have the same problem--for example, user A requested a new/pooled project, but the PI is also getting an email saying "Dear {first_name}, Your request to create/pool...".